contrib/qat/compression/qatzstd/compressor/source/BUILD

```diff
@@ -15,10 +15,20 @@ make(
     build_data = ["@com_github_qat_zstd//:all"],
     env = select({
         "//bazel:clang_build": {
-            "CFLAGS": "-Wno-error=unused-parameter -Wno-error=unused-command-line-argument -I$$EXT_BUILD_DEPS/qatlib/include -I$$EXT_BUILD_DEPS/zstd/include",
+            "CFLAGS": "-Wno-error=unused-parameter -Wno-error=unused-command-line-argument -I$$EXT_BUILD_DEPS/qatlib/include",
+            # Note: This path references the external directory directly. For bzlmod migration,
```

Comment:
lets remove these comments

there are many cases of using EXT_BUILD_ROOT and im about to add a load more - so repeating it here is kinda arbitrary